ci/src/main.rs

```diff
@@ -327,10 +327,8 @@ fn bump(version: &str, patch_bump: bool) -> String {
     }
     if major != 0 {
         format!("{}.0.0", major + 1)
-    } else if minor != 0 {
-        format!("0.{}.0", minor + 1)
     } else {
-        format!("0.0.{}", patch + 1)
+        format!("0.{}.0", minor + 1)
```

Comment:
Would this not preclude us from making patch releases until we have a 1.0?

Could you describe what problem this is addressing so I can understand the change?

Reply:
Oh never mind, I saw the `patch_bump` conditional above after expanding the diff further.

I still don't understand what this fixes, exactly.

Reply:
We want our "first" releases to be `v0.1.0`, not `v0.0.1` don't we?

Reply:
Oh right, for `wdl-engine` I thought we'd just manually bump it to `0.1.0`, but this works too!